collection/lscq/asm_arm64.s

```diff
@@ -0,0 +1,89 @@
+#include "textflag.h"
+#include "funcdata.h"
+
+TEXT ·compareAndSwapUint128(SB), NOSPLIT, $0-41
+	MOVD	addr+0(FP), R0
+	MOVD	old1+8(FP), R2
+	MOVD	old2+16(FP), R3
+	MOVD	new1+24(FP), R4
+	MOVD	new2+32(FP), R5
+	MOVBU	·arm64HasAtomics+0(SB), R1
+    CBZ     R1, load_store_loop
```

Comment:
Looks the codes here are not aligned, it would be nice if you can format this function :)

Reply:
Resolved. I've just add indention check for my IDE.